client/me/purchases/paths.js

```diff
@@ -26,7 +26,11 @@ function cancelPrivateRegistration( siteName, purchaseId ) {
 	return managePurchase( siteName, purchaseId ) + '/cancel-private-registration';
 }
 
-function editCardDetails( siteName, purchaseId, cardId = ':cardId' ) {
+function editCardDetails( siteName, purchaseId ) {
+	return editPaymentMethod( siteName, purchaseId ) + `/edit`;
+}
+
+function editCardDetailsWithCard( siteName, purchaseId, cardId = ':cardId' ) {
```

Comment:
Also super minor, but I think naming this something like `editSpecificCardDetails` would make it easier to understand the difference between it and the previous function.